### PR TITLE
chore: add split repo templates

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: google/cloud/vision_v1p1beta1/proto/image_annotator.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc.
+// Copyright 2019 Google LLC.
```
Comment (leahecole) [inline google/cloud/vision_v1p1beta1/proto/image_annotator.proto]: This probably should stay at 2017, right? Also, you'll have to take away the period after LLC for license bot to be happy

File: google/cloud/vision_v1p2beta1/proto/image_annotator.proto [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google Inc.
+// Copyright 2019 Google LLC.
```
Comment (leahecole) [inline google/cloud/vision_v1p2beta1/proto/image_annotator.proto]: same as other proto, should probably stay at 2017, remove period after LLC

File: google/cloud/vision_v1p3beta1/proto/image_annotator.proto [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google Inc.
+// Copyright 2019 Google LLC.
```
Comment (leahecole) [inline google/cloud/vision_v1p3beta1/proto/image_annotator.proto]: 2017, period after llc

File: google/cloud/vision_v1p3beta1/proto/product_search.proto [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google Inc.
+// Copyright 2019 Google LLC.
```
Comment (leahecole) [inline google/cloud/vision_v1p3beta1/proto/product_search.proto]: 2017, period after llc

**Comment by leahecole** [comment]
oops sometimes i said 2017 but it should be 2018. moral of story is year shouldn't change i don't think

**Comment by crwilcox** [comment]
I can revert those specific lines, but those are coming from generation. @busunkim96 what is the plan with these?

**Comment by busunkim96** [comment]
@crwilcox We're continuing to take the version bumps in Python until we are told to do otherwise. I don't have strong opinions about them going through in this PR or in a separate one

**Pull request merged**